src/HelixJobCreator.cs

```diff
@@ -106,8 +115,8 @@ public async Task<AgentInfoItem> CreateJob()
                     .WithFiles(credentialsPath, agentSettingsPath, StartupScriptPath)
                     .WithTimeout(TimeSpan.FromMinutes((double)_configuration.TimeoutInMinutes))
                     .AttachToJob()
-                    .SendAsync();
-
+                    .SendAsync(null, cancellationToken);
+                cancellationToken.ThrowIfCancellationRequested();
```

Comment:
This one probably isn't _super_ useful, since it just stops us from returning. :-)

Reply:
ah it was useful before because we'd quit early and return false.